<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>512</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background: #101920;}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 40px;
color: #FCE417;</string>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>263</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 16px;
/* identical to box height */


color: #FCE417;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="signing">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>429</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFE715;
border-radius: 30;
font-family: Open Sans;
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 33px;

color: #101920;
</string>
    </property>
    <property name="text">
     <string>Signing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyfile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFE715;
border-radius: 2px;
font-family: Open Sans;
font-style: normal;
font-weight: 600;
font-size: 12px;
color: #101920;</string>
    </property>
    <property name="text">
     <string>Select file to Verify</string>
    </property>
   </widget>
   <widget class="QLabel" name="key">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #ffffff;
border-radius: 2px;
font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 16px;
/* identical to box height */


color: #101920;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="selectkey">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFE715;
border-radius: 2px;
font-family: Open Sans;
font-style: normal;
font-weight: 600;
font-size: 12px;
color: #101920;</string>
    </property>
    <property name="text">
     <string>Select Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genKey">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFE715;
border-radius: 30;
font-family: Open Sans;
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 33px;

color: #101920;
</string>
    </property>
    <property name="text">
     <string>Generate 
Key</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="file">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 10px;
/* identical to box height */


color: #101920;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="warn_red">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 16px;
/* identical to box height */


color: red;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="warn_green">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">ont-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 16px;
/* identical to box height */


color: lime;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="digest">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 10px;
/* identical to box height */


color: #101920;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
